src/components/Films/Films.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, FC } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getFilmData } from '../Redux/Action';
+import { useState } from 'react';
```

Comment:
why this is not imported from import above import React, { useEffect, FC, useState }